src/schemas/21-22A/schema.js

```diff
@@ -2,51 +2,110 @@ import _ from 'lodash';
 import constants from '../../common/constants';
 import definitions from '../../common/definitions';
 
+export const address = {
+  type: 'object',
+  properties: {
+    addressLine1: { type: 'string', minLength: 1, maxLength: 30 },
+    addressLine2: { type: 'string', minLength: 1, maxLength: 5 },
+    city: { type: 'string', minLength: 1, maxLength: 18 },
+    state_code: { type: 'string', minLength: 2, maxLength: 2 },
+    country: { type: 'string', minLength: 3, maxLength: 3 },
```

Comment:
Should I import `enum`s of states and countries here?

Reply:
I wrote a custom address object because the backend uses `address_line1`, `2`, etc.  Would it make more sense to use the existing `hcaAddress` object and change the data handling on the backend?

What do you think @cosu419 ?

Reply:
Don't think the state/country enums are necessary. `hcaAddress` was copied from another schema and can probably be replaced.